### PR TITLE
update in Kitematic OS support

### PR DESCRIPTION
**Pull request by ch3lo**
Kitematic now support Windows

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -289,7 +289,7 @@ system
 * [Docker Compose](https://github.com/docker/compose) (formerly Fig): 
 Define and run multi-container apps
 * [Kitematic](https://github.com/kitematic/kitematic): The easiest way to use 
-Docker on a Mac
+Docker on a Mac and Windows
```
Comment (thaJeztah) [inline README.md]: Could you remove "a" here? (So `Docker on Mac and Windows`)

**Comment by thaJeztah** [comment]
Thanks for this, @ch3lo 

I added one small correction, LGTM if that is corrected

ping @fredlf @jamtur01 @SvenDowideit @moxiegirl

**Comment by jamtur01** [comment]
LGTM if corrected too.

File: README.md (hunk 1 above)
Comment (ch3lo) [inline README.md]: Done. Waiting for Jenkins build

**Comment by thaJeztah** [comment]
@ch3lo thanks for addressing, can you squash your commits so that there's only a single commit? You can find instructions for that here; https://docs.docker.com/project/work-issue/#pull-and-rebase-frequently

Let us know if you need help! :heart:

**Comment by SvenDowideit** [comment]
LGTM when squashed.

**Comment by thaJeztah** [comment]
Thank you, @ch3lo!

(janky is running)

**Pull request merged**